### PR TITLE
DP-1474-Add-Replace-Log4j-with-SLF4J-in-Maven-in-DP-Demos

### PR DESCRIPTION
**Pull request by Bryan-Noel**

### CODE REVIEW

**Review by DeleTaylor — CHANGES_REQUESTED** [review]
@Bryan-Noel A few things:
1. Please prefix the Maven and Gradle project names with `DataPipeline-Demo-`.  
2. Let's also rename their example classes to `ReplaceLog4jWithSlf4jInMaven` and `ReplaceLog4jWithSlf4jInGradle`.
3. Can you try writing to & reading from S3 to see if those libraries cause the problem we're trying to uncover?  (You can add it after the current example code.)

**Comment by Bryan-Noel** [comment]
> @Bryan-Noel A few things:
> 
>     1. Please prefix the Maven and Gradle project names with `DataPipeline-Demo-`.
> 
>     2. Let's also rename their example classes to `ReplaceLog4jWithSlf4jInMaven` and `ReplaceLog4jWithSlf4jInGradle`.
> 
>     3. Can you try writing to & reading from S3 to see if those libraries cause the problem we're trying to uncover?  (You can add it after the current example code.)

I have updated the examples based on these requests. I did not see the problems when reading and writing from S3 either.

**Review by DeleTaylor — APPROVED** [review]

File: DataPipeline-Demo-Replace-Log4j-with-SLF4J-in-Maven/pom.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+    </properties>
+
+    <repositories>
+        <repository>
+            <id>datapipeline-repo</id>
+            <url>https://northconcepts.mycloudrepo.io/public/repositories/datapipeline/</url>
```
Comment (DeleTaylor) [inline DataPipeline-Demo-Replace-Log4j-with-SLF4J-in-Maven/pom.xml]: Can we use `maven.northconcepts.com` everywhere?

**Review by DeleTaylor — CHANGES_REQUESTED** [review]

File: DataPipeline-Demo-Replace-Log4j-with-SLF4J-in-Maven/pom.xml (hunk 1 above)
Comment (Bryan-Noel) [reply DataPipeline-Demo-Replace-Log4j-with-SLF4J-in-Maven/pom.xml]: done

**Review by DeleTaylor — APPROVED** [review]

**Pull request merged**